cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -49,20 +49,20 @@ def config_manager(config_manager_obj):
 @pytest.fixture(scope="function")
 def catalog_item(request, config_manager, dialog, catalog):
     config_manager_obj = config_manager
+    provider_name = config_manager_obj.yaml_data.get('name')
     provisioning_data = config_manager_obj.yaml_data['provisioning_data']
-    item_type, provider_type, provider, template = map(provisioning_data.get,
-                                                       ('item_type',
-                                                        'provider_type',
-                                                        'provider',
-                                                        'template'))
+    item_type, provider_type, template = map(provisioning_data.get,
+                                            ('item_type', 'provider_type', 'template'))
     item_name = dialog.element_data.get("default_text_box")
     catalog_item = CatalogItem(item_type=item_type,
                                name=item_name,
                                description="my catalog",
                                display_in=True,
                                catalog=catalog,
                                dialog=dialog,
-                               provider=provider,
+                               provider=version.pick({
+                                   '5.7': provider_name + ' Configuration Manager',
```

Comment:
Use a consistent syntax, this is the same exact message as printed in the quad_name property above, I would stick with the string.format() syntax.

If the 'name' attribute isn't found, `provider_name` will be None, and this will raise exception for invalid operand